pkg/controllers/offlineupgrade_reconcile.go

```diff
@@ -217,7 +223,7 @@ func (o *OfflineUpgradeReconciler) checkForNewPods(ctx context.Context) (ctrl.Re
 	}
 	if !foundPodWithNewImage {
 		o.Log.Info("Requeue to wait until at least one pod exists with the new image")
-		return ctrl.Result{Requeue: true}, nil
+		return ctrl.Result{Requeue: true, RequeueAfter: time.Duration(o.Vdb.Spec.UpgradeRequeueTime)}, nil
```

Comment:
We shouldn't need this.  The one added in the Reconcile function should handle adding the RequeueAfter time.

Reply:
Just saw this, added in the new commit.